packages/web/localizations/en.json

```diff
@@ -77,7 +77,7 @@
     "maxPrice": "Max price",
     "minPrice": "Min price",
     "collectRewards": "Collect Rewards",
-    "noRewards": "No rewards to collect",
+    "noRewards": "Nothing to collect, check back soon",
```

Comment:
why aren't the other languages updated?

Reply:
I'm waiting until the end to generate the remaining langs via CLI. There may be more copy changes to make